src/BugsnagUnity/Native/Fallback/NativeClient.cs

```diff
@@ -12,18 +13,42 @@ class NativeClient : INativeClient
 
         public IDelivery Delivery { get; }
 
-        private Dictionary<string, object> _editorMetadata = new Dictionary<string, object>();
+        private Dictionary<string, object> _fallbackMetadata = new Dictionary<string, object>();
+
+        private bool _launchMarkedAsCompleted = false;
+
+        private bool _hasRecivedLowMemoryWarning = false;
 
         public NativeClient(IConfiguration configuration)
         {
             Configuration = configuration;
             Breadcrumbs = new Breadcrumbs(configuration);
             Delivery = new Delivery();
+            Application.lowMemory += () => { _hasRecivedLowMemoryWarning = true; };
```

Comment:
Is there any way of detecting when the app exits the lowMemory state, or is this not possible with the available APIs?

Reply:
Not possible unfortunatly